traitsui/qt4/file_editor.py

```diff
@@ -110,18 +111,11 @@ def show_file_dialog(self):
         # file name is to be used (ie. an existing one to be opened or a new
         # one to be created).
         dlg = self._create_file_dialog()
+        dlg.open()
 
-        if dlg.exec_() == QtGui.QDialog.DialogCode.Accepted:
-            files = dlg.selectedFiles()
-
-            if len(files) > 0:
-                file_name = str(files[0])
-
-                if self.factory.truncate_ext:
-                    file_name = splitext(file_name)[0]
```

Comment:
seems like we no longer truncate the extension?

Reply:
I missed that - that should be put back in.